src/graphorm/index.ts

```diff
@@ -119,6 +120,38 @@ const obj = new GraphORM({
         },
         transform: (value: number): boolean => value > 0,
       },
+      companies: {
+        relation: {
+          isMany: true,
+          customRelation: (ctx, parentAlias, childAlias, qb): QueryBuilder =>
+            qb
+              .innerJoin(
+                UserCompany,
+                'uc',
+                `"${childAlias}"."id" = uc."companyId"`,
+              )
+              .where('uc.verified = true')
+              .andWhere(`uc."userId" = "${parentAlias}".id`),
```

Comment:
```suggestion
              .andWhere(`uc."userId" = "${parentAlias}".id`)
              .limit(50),
```

just so we don't return hundreds of rows if somebody tries to be naughty, we do the same for `privilegedMembers`

Reply:
For now you can actually only add 1 anyway, we dont support more from fe.

We can tackle this maybe in follow up once we ever do?

Reply:
But from API directly someone could do it 😆

Reply:
Someone naughty xD